test/mocha.opts

```diff
@@ -1,3 +1,2 @@
 --ui tdd
 --require source-map-support/register
-lib/test/*_test.js lib/test/**/*_test.js
```

Comment:
Or, hm, rather I think this should be put into the package.json

Though I've had good luck lately with using `.only`, it seems mocha fixed the bugs I was seeing with it at some point.